### DISCUSSION TITLE
Include API Provider and Model info in the history

### DISCUSSION THREAD

**Discussion by Freffles** [cline/cline#610] [Feature Requests]
I would like to see the API provider and the Model identified in the history export. I find myself changing models as I'm learning to use Cline and it would he handy to look at the history and understand what model was being used when things went right or went wrong. It would also be nice to have the tokens information that is displayed on the task. Input Tokens, Input Tokens, Cache usage and API cost.
![image](https://github.com/user-attachments/assets/771523d5-3425-4e07-9991-3cdfcdbc76a9)

**Comment by JHON12091986** [comment]
Hi, I'd like to work on this feature request. I can add the API provider, model, and token usage information to the history export in Cline.

My approach:
1. Identify where history data is stored and exported
2. Add fields for API provider, model, input tokens, output tokens, cache usage, and cost
3. Ensure backward compatibility
4. Add tests

Please assign this issue to me.

**Comment by JHON12091986** [comment]
Hi, I'm interested in working on this feature request. Could someone please assign it to me? I've already outlined my approach in the comments.
